pallets/api/src/messaging/mod.rs

```diff
@@ -93,33 +94,48 @@ pub mod pallet {
 		/// request.
 		#[pallet::constant]
 		type MaxContextLen: Get<u32>;
+
+		/// SAFETY: should be less than or equal to u16.
 		/// The maximum length of outbound (posted) data.
 		#[pallet::constant]
 		type MaxDataLen: Get<u32>;
+		
+		/// SAFETY: should be less than or equal to u16.
+		/// The maximum amount of key for an outbound request.
 		#[pallet::constant]
 		type MaxKeys: Get<u32>;
+		
+		/// SAFETY: should be less than or equal to u16.
+		/// The maximum byte length for a single key of an ismp request.
 		#[pallet::constant]
 		type MaxKeyLen: Get<u32>;
-
+		
+		/// The maximum lenght for a response.
```

Comment:
nit: typo in length.

Reply:
might be good to add a spell-check extension to VSCode. Rust Rover highlights these in red for me so I often point them out :)